server/routes/manage-csips/view.njk

```diff
@@ -0,0 +1,168 @@
+{% from "partials/simplePagination/macro.njk" import simplePagination %}
+{% from "partials/statusTag/macro.njk" import statusTag %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "./components/reviewDate.njk" import reviewDate %}
+
+{% set fullWidth = true %}
+{% set pageTitle = "View and manage CSIPs" %}
+{% extends "partials/layout.njk" %}
+
+{% block innerContent %}
+  <h1 class="govuk-heading-xl govuk-!-margin-bottom-6">View and manage CSIPs </h1>
+
+  <form method="post" class="govuk-grid-row govuk-!-margin-bottom-7 govuk-!-padding-6 search-form">
+    <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+    <h2 class="govuk-heading-m">Filter by</h2>
+    <div class="horizontal-form">
+      {{ govukInput({
+        id: "query",
+        name: "query",
+        value: query,
+        classes: "min-width-one-quarter",
+        label: {
+          text: 'Name or prison number'
+        }
+      }) }}
+
+      {{ govukSelect({
+        id: "status",
+        name: "status",
+        label: {
+          text: "CSIP status"
+        },
+        items: [
+          {
+            value: "",
+            text: "All"
+          },
+          {
+            value: 'AWAITING_DECISION',
+            text: 'Awaiting decision'
+          },
+          {
+            value: 'CSIP_CLOSED',
+            text: 'CSIP closed'
+          },
+          {
+            value: 'CSIP_OPEN',
+            text: 'CSIP open'
+          },
+          {
+            value: 'INVESTIGATION_PENDING',
+            text: 'Investigation pending'
+          },
+          {
+            value: 'NO_FURTHER_ACTION',
+            text: 'No further action'
+          },
+          {
+            value: 'PLAN_PENDING',
+            text: 'Plan pending'
+          },
+          {
+            value: 'REFERRAL_PENDING',
+            text: 'Referral pending'
+          },
+          {
+            value: 'REFERRAL_SUBMITTED',
+            text: 'Referral submitted'
+          },
+          {
+            value: 'SUPPORT_OUTSIDE_CSIP',
+            text: 'Support outside of CSIP'
+          },
+          {
+            value: 'ACCT_SUPPORT',
+            text: 'Support through ACCT'
+          }
+        ] | setSelected(status or '')
+      }) }}
+
+      {{ govukButton({
+        text: "Apply filters",
+        preventDoubleClick: true
+      }) }}
+      <a class="govuk-link govuk-link--no-visited-state govuk-body no-label-form-clear-link" href="?clear=true">Clear</a>
```

Comment:
Dont think you need `=true` - the presence of the query param should be enough?

Reply:
clear=any_value would work. just having ?clear or empty ?clear= wouldn't work because i'm only checking `if (clear)` not `if (clear !== undefined)`